apps/webapp/src/components/swap/swap-loader.tsx

```diff
@@ -26,13 +31,18 @@ export interface SwapLoaderResponse {
 const getAndSetDefaultAssetBalances = async () => {
   const assetBalances = await getBalances();
 
+  // filter assets that are not available for swap
+  const filteredAssetBalances = assetBalances.filter(b =>
+      [assetPatterns.lpNft, assetPatterns.proposalNft, assetPatterns.unbondingToken, assetPatterns.votingReceipt]
```

Comment:
I think there could potentially be a market for unbonding tokens. Think we should keep that one.

Reply:
It's actually kind of weird. 
We will have 100 different liquid staking tokens for 100 different validators, and for each of these liquid staking tokens we can have many more unbonding tokens for each epoch  :exploding_head:

Reply:
But in general I agree, removed unbonding tokens from filtering